DATA-1902 Adding Question warning message

madmaxmatze opened the pull request:
https://machtfit.atlassian.net/browse/DATA-1902

Jyggafey commented on mf-script.js:
```diff
@@ -0,0 +1,109 @@
+        `;
+
+        // Append the notice to the body if it's not already present
+        if (!document.querySelector('.machtfit-big-red-notice')) {
```
selector doesnt match className above

madmaxmatze committed:
question notice added

Jyggafey committed:
set location of mf-style.css

Use the path of the file in github pages, in order to simplify deployment via reverse proxy to github pages.
